packages/block-library/src/query/index.php

```diff
@@ -31,8 +31,17 @@ function render_block_core_query( $attributes, $content, $block ) {
 			// Add the necessary directives.
 			$p->set_attribute( 'data-wp-interactive', 'core/query' );
 			$p->set_attribute( 'data-wp-router-region', 'query-' . $attributes['queryId'] );
-			$p->set_attribute( 'data-wp-context', '{}' );
+			$p->set_attribute(
+				'data-wp-context',
+				wp_json_encode(
+					[
+						'routerRegion' => 'query-' . $attributes['queryId'],
+						'isLoading'    => false,
+					]
+				)
+			);
 			$p->set_attribute( 'data-wp-key', $attributes['queryId'] );
+			$p->set_attribute( 'data-wp-class--is-loading', 'context.isLoading' );
```

Comment:
Same here: `data-wp-class--is-current-navigation-origin`

Reply:
Fixed in [d037eb7](https://github.com/WordPress/gutenberg/pull/67820/commits/d037eb723feba4aaae3d1407c15631a7e9697e02)